content/04.use-cases-b.md

```diff
@@ -78,16 +78,36 @@ Periodic code review can also help advisors to identify errors early in the proc
 
 ### Writing manuscript
 
-*contributors to this section:*\
-Caveat that GitHub has been called out for not being so user-friendly for manuscript development (Ram 2013).
-But getting better?
-Tools that link with GitHub have been developed with synchronous writing in mind.
-HackMD provides a collaborative writing platform based on Markdown that integrates with GitHub.\
-We used this platform early on in the process of writing this manuscript to generate an outline.
+<!--*contributors to this section:* Emma J. Hudgins-->
 
-Later, we wrote the manuscript by Manubot, which is one of the important collaborative manuscript platform uses Markdown for writing and GitHub for storing and tracking changes over time [@https://doi.org/10.1371/journal.pcbi.1007128]
+Beyond supporting collaboration at the level of code, GitHub can even be used for collaborative writing of manuscripts. 
+Writing a manuscript in GitHub and storing it with the data and code associated with a repository allows for all files associated with a research product to be found in one place.
+Further, authors can make use of the Discussions tab to suggest relevant papers to be cited, and can raise issues during the writing process that can be assigned to collaborators.
+
+While GitHub is not considered as user-friendly for manuscript development as conventional word processors (Ram 2013), it has been substantially improved with recent tools.
+An important collaborative tool is [HackMD](https://hackmd.io), allows real-time collaboration like Google Docs for individual Markdown documents. 
```

Comment:
```suggestion
An important collaborative tool is [HackMD](https://hackmd.io), which allows real-time collaboration like Google Docs for individual Markdown documents. 
```

Have we explained what Markdown is at this point in the manuscript?  If not, I think the first point to make is that although GitHub doesn't work well with Word, it _does_ work well with Markdown which can be rendered to Word, PDF, and HTML.